packages/typespec-python/src/emitter.ts

```diff
@@ -1128,7 +1135,9 @@ function emitServerParams(context: SdkContext, namespace: Namespace): Record<str
                 param: param,
             };
             const emittedParameter = emitParameter(context, serverParameter, "Client");
-            endpointPathParameters.push(emittedParameter);
+            if (!endpointPathParameters.some((p) => p.clientName === emittedParameter.clientName)) {
+                endpointPathParameters.push(emittedParameter);
+            }
```

Comment:
To avoid duplicated endPointPath: https://github.com/Azure/autorest.python/blob/81195528ac6b1bb4b24db45cebffbd508df3a35e/packages/typespec-python/test/generated/client-structure-multiclient/client/structure/multiclient/_operations/_operations.py#L299-L300